src/Grid/save_the_grids.jl

```diff
@@ -0,0 +1,213 @@
+	close(file)
+end
+
+export save_the_grid
+
+function toml_saves_the_grids(grid::Grid,accuracy::Integer=8)
+	t_string::String = ""
+
+	t_string *= toml_saves_the_grids(grid.nodes)
+	t_string *= toml_saves_the_grids(grid.nodesets,"Nodesets")
+	t_string *= toml_saves_the_grids(grid.cellsets,"Cellsets")
+	t_string *= toml_saves_the_grids(grid.vertexsets)
+	t_string *= toml_saves_the_grids(grid.facetsets)
+	t_string *= toml_saves_the_grids(grid.cells)
+
+	return t_string
+end
+
+function toml_saves_the_grids(cells::AbstractVector{<:Ferrite.AbstractCell})
+	t_string::String = ""
+	for cell in cells
+		t_string *= @sprintf("%s",toml_saves_the_grids(cell))
+	end
+
+	return t_string
+end
+
+function toml_saves_the_grids(cell::Ferrite.AbstractCell{rshape}) where {rshape}
+	t_string::String = "[[Cells]]\n"
+	t_string *= @sprintf("type = \"%s\"\n",typeof(cell))
+	t_string *= @sprintf("nodes = %s\n",toml_saves_the_grids(cell.nodes))
+	t_string *= "\n"
+
+	return t_string
+end
+
+function toml_saves_the_grids(tpl::NTuple{N,<:Integer}) where N
+	t_string::String = "[ "
+
+	for n in 1:N
+		t_string *= @sprintf("%d,",tpl[n])
+	end
+
+	t_string = t_string[1:end-1]
+
+	t_string *= " ]"
+end
+
+function toml_saves_the_grids(facetsets::Dict{String,OrderedSet{FacetIndex}})
+	t_string::String = "[Facetsets]\n"
+
+	for (setname,facetset) in facetsets
+		t_string *= @sprintf("%s = %s\n",setname,toml_saves_the_grids(facetset))
+	end
+
+	t_string *= "\n"
+
+	return t_string
+end
+
+function toml_saves_the_grids(facetset::OrderedSet{FacetIndex})
+	t_string::String = "[ "
+
+	for facet in facetset
+		t_string *= @sprintf("[%d,%d],",facet[1],facet[2])
+	end
+
+	t_string = t_string[1:end-1]
+
+	t_string *= " ]"
+
+	return t_string
+end
+
+# used for nodesets and cellsets
+function toml_saves_the_grids(sets::Dict{String,OrderedSet{Int64}},collection_name::String)
+	t_string::String = @sprintf("[%s]\n",collection_name)
+
+	for (setname,cellset) in sets
+		t_string *= @sprintf("%s = %s\n", setname, toml_saves_the_grids(cellset))
+	end
+
+	t_string *= "\n"
+
+	return t_string
+end
+
+function toml_saves_the_grids(intset::OrderedSet{<:Integer})
+	t_string::String = "[ "
+
+	for x in intset
+		t_string *= @sprintf("%d,",x)
+	end
+
+	t_string = t_string[1:end-1]
+
+	t_string *= " ]"
+
+	return t_string
+end
+
+function toml_saves_the_grids(nodes::AbstractVector{<:Node},accuracy::Integer=8)
+	t_string::String = "[Nodes]\n"
+	t_string *= "nodes = [ "
+
+	for node in nodes
+		t_string *= @sprintf("%s,",toml_saves_the_grids(node))
+	end
+
+	t_string = t_string[1:end-1]
+	
+	t_string *= " ]\n"
+	t_string *= "\n"
+
+	return t_string
+end
+
+function toml_saves_the_grids(node::Node{dim}) where dim
+	t_string::String = "["
+
+	for i in 1:dim
+		t_string *= @sprintf("%s,",toml_saves_the_grids(node.x[i]))
+	end
+
+	t_string = t_string[1:end-1]
+
+	t_string *= "]"
+
+	return t_string
+end
+
+# find a small number of digits for precise storage
+function toml_saves_the_grids(x::AbstractFloat)
```

Comment:
I am not confident that this works properly for all float types. Seems like this implementation is specific to Float64 and (more narrow types). But this function essentially generates a string of the truncated real number represenation?